src/Inciter/Partitioner.h

```diff
@@ -1270,6 +1276,7 @@ class Partitioner : public CBase_Partitioner< HostProxy,
     }
     //! \brief Compute standard deviation of the communication cost of merging
     //!   the linear system
+    //! \param[in] host Host to signal to
```

Comment:
Are these computation tasks or wrappers for charm messages? Do we want to couple our compute so closely with our communication?